src/mpl/src/shm/mpl_shm.c

```diff
@@ -53,9 +51,8 @@ int MPL_shm_hnd_deserialize(MPL_shm_hnd_t hnd, const char *str_hnd, size_t str_h
 
 int MPL_shm_hnd_get_serialized_by_ref(MPL_shm_hnd_t hnd, char **str_ptr)
 {
-    int mpi_errno = 0;
     *str_ptr = (char *) MPLI_shm_ghnd_get_by_ref(hnd);
-    return mpi_errno;
+    return 0;
 }
```

Comment:
Should this change go to commit: mpl/shm: Remove unnecessary variables for return ?